tests/simulation/test_simulator.py

```diff
@@ -151,3 +166,22 @@ def test_new_syncing_peer(self):
             self.log.debug(f'checking node {idx}')
             self.assertConsensusValid(node)
             self.assertConsensusEqual(node, late_manager)
+
+
+class SyncV1RandomSimulatorTestCase(unittest.SyncV1Params, BaseRandomSimulatorTestCase):
+    __test__ = True
+
+    # XXX: I'm not sure if marking this as flaky is ideal, but it can fail randomly if it lands on a bad seed,
+    #      known bad seeds are saved bellow, failing twice in a row is still possible but much less likely
+    @pytest.mark.flaky(max_runs=2, min_passes=1)
```

Comment:
I think we should remove this and let them fail. This is a simple test and I feel we can debug it quickly.

Reply:
Done.

Reply:
Great. Thanks!